content/support-training.html

```diff
@@ -0,0 +1,173 @@
+                <h3>Learn how to Monitor with Prometheus</h3>
+              </a>
+              <p>A set of courses. Master master Prometheus, including the data model, PromQL, configuration, Grafana, alerting, instrumenting and practical deployment.</p>
+              <p>Created by Brian Brazil and delivered via <a href="https://training.robustperception.io/" target="_blank">Robust Perception Training</a>.
+            </div>
+          </div>
+      </div>
+      <div class="col-sm-12 col-md-6 doc-content panel-col">
+          <div class="panel panel-default">
+            <div class="panel-body">
+              <a href="https://training.linuxfoundation.org/training/monitoring-systems-and-services-with-prometheus-lfs241/" target="_blank">
+                <h3>Monitoring Systems and Services with Prometheus</h3>
+              </a>
+              <p>This course leads new Prometheus users through many of its major features, best practices, and use cases. Course participants are expected to have basic experience with 
+                Linux/Unix system administration, as well as some development experience in Go and/or Python.</p>
+                <p>Created by Julius Volz and delivered via <a href="https://training.linuxfoundation.org/" target="_blank">Linux Foundation Training</a></p>
+
+            </div>
+          </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row gray-bg">
+    <div class="container">
+      <div class="col-md-12 doc-content training-community">
+        <h2>Commercial support</h2>
+        <p>
+          These resources are created by members of the Prometheus community who are 
+          not directly affiliated with the project. This list is provided in alphabetical order.
+        </p>
+      </div>
+    </div>
+    <div class="container equal">
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.container-solutions.com/" class="btn btn-training btn-default darkgray-bg btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/container-solutions.svg" alt="Container Solutions">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.credativ.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/credativ.png" alt="Credativ">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://grafana.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/grafana-labs.svg" alt="Grafana Labs">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://juliusv.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            Julis Volz <br/> 
```

Comment:
"Julis" -> "Julius" :)

Reply:
I took the weekend off after submitting this, clearly with good cause 🤦‍♀️